pkg/monitoring/helper_test.go

```
@@ -29,11 +32,13 @@ func generateChainConfig() CosmosConfig {
 	}
 }
 
-func generateFeedConfig() CosmosFeedConfig {
+func generateFeedConfig(t *testing.T) CosmosFeedConfig {
 	coins := []string{"btc", "eth", "matic", "link", "avax", "ftt", "srm", "usdc", "sol", "ray"}
 	coin := coins[rand.Intn(len(coins))]
-	address, _ := sdk.AccAddressFromBech32("wasm106x8mk9asfnptt5rqw5kx6hs8f75fseqa8rfz2")
-	proxyAddress, _ := sdk.AccAddressFromBech32("wasm106x8mk9asfnptt5rqw5kx6hs8f75fseqa8rfz2")
```

These unchecked errors were leading to silent failures